renderer.go

```diff
@@ -125,3 +140,8 @@ func (r *Renderer) RenderHeader(w io.Writer, ast *bf.Node) {
 func (r *Renderer) RenderFooter(w io.Writer, ast *bf.Node) {
 	r.Base.RenderFooter(w, ast)
 }
+
+// ChromaCSS returns CSS used with chroma's html.WithClasses() option
+func (r *Renderer) ChromaCSS(w io.Writer) {
+	r.Formatter.WriteCSS(w, r.Style)
+}
```

Comment:
I'd prefer if the error could be returned here

Reply:
In principle, yes, and I agree, but blackfriday itself is not bothering to return error (which also is weird...) so I just left it off for consistency. Up to you, I can add it.

The function that actually writes CSS only returns errors when there is a `Fprintf` error IIRC so not like there is high risk of something going wrong but I guess upstream implementation can always change.

Reply:
From what I understand, this function is exposed and thus the user should be able to handle this error, even though it's unlikely there would be one. I prefer to delegate the handling of this "unlikely" error to the end user and not fail silently. 
(Even though I agree, it's quite weird that there's no proper way of returning an error with BF)